BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -96,6 +96,8 @@ public class BDACompetitionMode : MonoBehaviour
         GUIStyle dateStyleShadow;
         Rect dateRect;
         Rect dateRectShadow;
+        Rect versionRect;
```

Comment:
Following the confusion during the S5R-1 stream on whether or not Splag had the proper patched .dll or not, SJ asked for an easy version checker